content/release-notes/1.19.3.md

```diff
@@ -0,0 +1,10 @@
+---
+date: 2020-09-18
+linktitle: "1.19.3"
+title: "1.19.3"
+kubernetes: "1.16, 1.17, and 1.18"
+---
+
+{{<changes>}}
+* Removed support for TLS versions below 1.2 on embedded cluster installations.
```

Comment:
That makes it sound negative. Can we flip it and say something positive? Like enforce higher security while removing deprecated versions?

Reply:
Sure, I'll make a new PR